src/components/CorporateCampaign/CampaignLoanRow.vue

```diff
@@ -36,7 +36,7 @@
 						:item-index="index"
 						:key="`loan-${loan}`"
 						:loan-id="loan"
-						:lend-now-button="true"
+						:show-action-button="true"
```

Comment:
Do we want the new loan card or are we going to continue using this older variation?

Reply:
I think we're going to move forward with using the older variation until the Denali card exp is fully complete and I'll reach back out with the team to switch it out! :)